src/cpu/x64/jit_avx512_core_bf16_conv_kernel.hpp

```diff
@@ -238,7 +242,7 @@ struct jit_avx512_core_bf16_fwd_kernel {
         return status::out_of_memory;
     }
 
-    ~jit_avx512_core_bf16_fwd_kernel() { delete kernel_; }
+    ~jit_avx512_core_bf16_fwd_kernel() {}
```

Comment:
would like to delete this entirely someday.

Reply:
this should be possible now, since `~jit_avx512_core_amx_bwd_data_kernel_t()` is deleted above

Reply:
or clang-tidy can do the cleanup!